airflow/utils/log/logging_mixin.py

```diff
@@ -109,7 +110,10 @@ def _propagate_log(self, message):
         """
         Propagate message removing escape codes.
         """
-        self.logger.log(self.level, remove_escape_codes(message))
+        if conf.getboolean('core', 'colored_ui_logs'):
+            self.logger.log(self.level, message)
+        else:
+            self.logger.log(self.level, remove_escape_codes(message))
```

Comment:
If ASCI codes are handled properly in WebUI I don't think we need this case?

Reply:
This is an option to the user, in the default case this new feature will be disabled, so we should ignore the ASCI carracters

Reply:
Got it. I am just wondering if colored logs in ui shouldn't be default. @potiuk @mik-laj WDYT?

Reply:
Stackdriver doesn't support colored logs. Many other external task handlers also. This can also be problematic for other handlers. We'd have to test it with all the handles to make sure we can turn it on by default.

Reply:
ElasticSearch is the same -- it would make searching more tricky if ANSI escape sequences got fed in to ES.